ci-operator/step-registry/cucushift/installer/check/control-plane-machinesets/cucushift-installer-check-control-plane-machinesets-commands.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+    # shellcheck disable=SC1090
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+
+# control-plane machinesets
+stderr=$(mktemp)
+stdout=$(mktemp)
+oc get controlplanemachinesets -n openshift-machine-api 1>${stdout} 2>${stderr} || true
+
+if [ -s ${stderr} ]; then
+    echo "WARN: 'controlplanemachinesets' not found, skip checking"
+    cat "${stderr}"
```

Comment:
I think we can print all `stderr` and `stdout` content before exit, it's useful for debugging.